### ISSUE TITLE
Workspace.getActiveEditor is deprecated.

### ISSUE THREAD

**Issue by luisrudge** [abe33/atom-bezier-curve-editor#9]
Call ::getActiveTextEditor instead
```
Workspace.getActiveEditor (C:\Users\luis\AppData\Local\atom\app-0.200.0\resources\app.asar\src\workspace.js:1022:12)
Object.replaceMatch (C:\Users\luis\.atom\packages\bezier-curve-editor\lib\bezier-curve-editor.coffee:106:28)
<unknown> (C:\Users\luis\.atom\packages\bezier-curve-editor\lib\bezier-curve-editor.coffee:25:7)
Emitter.emit (C:\Users\luis\AppData\Local\atom\app-0.200.0\resources\app.asar\node_modules\event-kit\lib\emitter.js:82:11)
Object.__NONE__ (C:\Users\luis\.atom\packages\bezier-curve-editor\lib\bezier-curve-editor-element.coffee:69:15)
HTMLButtonElement.listener (C:\Users\luis\.atom\packages\bezier-curve-editor\node_modules\atom-utils\lib\mixins\events-delegation.js:108:41)
```

**Comment by abe33** [comment]
Thanks for the feedback, I fixed this in v0.7.1.

**Comment by luisrudge** [comment]
Thaaaanks :)
